openmoc/plotter.py

```diff
@@ -1015,10 +1015,10 @@ def plot_spatial_data(domains_to_data, plot_params, get_figure=False):
     Examples
     --------
     A user may invoke this function from an Python file as follows:
```

Comment:
Can you fix this typo from before here and elsewhere: "...from an Python file" -> "...from a Python file..."